src/wikigen/VarGenerator.java

```diff
@@ -54,6 +54,8 @@ public void setup(String[] args){
                 Jval json = Jval.read(response.getResultAsString());
                 String latestRelease = json.asArray().first().getString("tag_name").substring(1);
                 out.put("latestRelease", latestRelease);
+                String latestReleaseLink = json.asArray().first().getString("html_url").substring(1);
```

Comment:
are you sure `.substring(1)` is necessary?

Reply:
@Anuken just realised it isnt lmao
that would break the link